vagrant/Vagrantfile

```diff
@@ -55,6 +65,8 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   # Every Vagrant development environment requires a box. You can search for
   # boxes at https://atlas.hashicorp.com/search.
   config.vm.box = $vm_box
+  # Execute command to install the plugin: "vagrant plugin install vagrant-disksize"
+  config.disksize.size = '50GB'
```

Comment:
Use `$vm_device_size` instead of hardcoding the value? You could bump the default if 20Gb isn't enough.